Sources/Private/Model/Objects/Transform.swift

```diff
@@ -13,10 +13,8 @@ final class Transform: Codable {
   // MARK: Lifecycle
 
   required init(from decoder: Decoder) throws {
-    /**
-     This manual override of decode is required because we want to throw an error
-     in the case that there is not position data.
-     */
+    /// This manual override of decode is required because we want to throw an error
+    /// in the case that there is not position data.
```

Comment:
```suggestion
    // This manual override of decode is required because we want to throw an error
    // in the case that there is not position data.
```